### PR TITLE
docs: waypoint download page changes

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## "Ready for Review" checklist

This PR updates the Waypoint download links. 

### Before
image

### After
image

- [x] The Vercel preview link has been added to this PR's description
- [x] The Asana task has been added to this PR's description
- [x] This PR's link has been added to the "📐 Pull Request" field on the Asana task
- [x] This Vercel preview link has been added to the "🔍 Preview" field on the Asana task
- [x] The description template has been filled in below

---

## Relevant links

- [Preview link](https://dev-portal-ea31h2dbd-hashicorp.vercel.app/waypoint/downloads) 🔎
- [Asana task](https://app.asana.com/0/1201979547919485/1201979547919574) 🎟️

## What

These changes are the result of a Dev Portal review asked from the Education team.

## Why

I changed the tutorials cards to focus on two themes. Getting started, and cloud providers. 
The first row focuses on the generic getting started, and the Kubernetes, and Nomad collection. I removed the Install, as it is redundant with the existing information on the page. The second row targets the cloud providers, which are more common usecases and more relevant to the practittioner.

### CODE REVIEW

**Comment by BrandonRomano** [comment]
Brilliant!  I am not sure why the deployment failed... I will ask my team 🤔

**Review by BrandonRomano — APPROVED** [review]
Okey I just rebased, I believe that will fix our build error.

I defer to @tunzor for content review side of things.

**Review by tunzor — COMMENTED** [review]
Good choices on the featured tutorials!
Minor copy/paste typo but otherwise LGTM

File: src/data/waypoint-install.json [hunk 1]
```diff
@@ -6,29 +6,29 @@
...
+      "description": "Use your existing AWS environment and leverage Waypoint to deploy applications to AWS ECS and AWS Lambda.",
+      "href": "https://learn.hashicorp.com/collections/waypoint/deploy-aws"
     },
     {
-      "title": "Get Started - Kubernetes",
-      "description": "Build, deploy, and release applications to a Kubernetes cluster.",
-      "href": "https://learn.hashicorp.com/collections/waypoint/get-started-kubernetes"
+      "title": "Deploy applications to Azure",
+      "description": "Learn how to use Waypoint to deploy applications to Azure Container Instances.",
+      "href": "https://learn.hashicorp.com/collections/waypoint/get-started-nomad"
```
Comment (tunzor) [inline src/data/waypoint-install.json:26]:
```suggestion
      "href": "https://learn.hashicorp.com/collections/waypoint/deploy-azure"
```

**Review by tunzor — CHANGES_REQUESTED** [review]

File: src/data/waypoint-install.json [hunk 2]
```diff
@@ -6,29 +6,29 @@
       "href": "https://learn.hashicorp.com/tutorials/waypoint/get-started-intro"
     },
     {
-      "title": "Install Waypoint",
-      "description": "Install the Waypoint binary locally on Mac, Linux, or Windows, manually or using a package manager (Homebrew).",
-      "href": "https://learn.hashicorp.com/tutorials/waypoint/get-started-install"
+      "title": "Get Started - Kubernetes",
+      "description": "Learn how you can use Waypoin to deploy application to a Kubernetes cluster.",
```
Comment (ashleemboyer) [inline src/data/waypoint-install.json:10]:
```suggestion
      "description": "Learn how you can use Waypoint to deploy application to a Kubernetes cluster.",
```

Comment (karl-cardenas-coding) [reply src/data/waypoint-install.json]: Nice catch @ashleemboyer :)

**Pull request merged**